### PR TITLE
abcde: fix flac bin output

### PR DESCRIPTION
**Pull request by Profpatsch**
The `flac` dependency in `abcde` used the wrong output and the binary didn’t exist.

### CODE REVIEW

**Comment by groxxda** [comment]
I'd rather use `lib.makeBinPath [ pkgs.. ]` here instead of hardcoding the output names and appending `/bin` to them.

**Comment by Profpatsch** [comment]
I think that’s purely cosmetic, but will integrate it.

**Comment by Profpatsch** [comment]
Done.

File: pkgs/applications/audio/abcde/default.nix [hunk 1]
```diff
@@ -52,7 +52,7 @@ in
          --replace '#!/usr/bin/perl' '#!${perl}/bin/perl'
 
       wrapProgram "$out/bin/abcde" --prefix PATH ":" \
-        "$out/bin:${which}/bin:${libcdio}/bin:${cddiscid}/bin:${wget}/bin:${vorbis-tools}/bin:${id3v2}/bin:${eyeD3}/bin:${lame}/bin:${flac.bin}/bin"
+        ${stdenv.lib.makeBinPath [ "$out" which libcdio cddiscid wget vorbis-tools id3v2 eyeD3 lame flac.bin ]}
```
Comment (groxxda) [inline pkgs/applications/audio/abcde/default.nix]: no need for `.bin` anymore with `makeBinPath`

Comment (Profpatsch) [inline pkgs/applications/audio/abcde/default.nix]: Oh, that’s cool. I see now.

Comment (Profpatsch) [inline pkgs/applications/audio/abcde/default.nix]: Done.

**Pull request merged**